serialization-jackson/src/main/resources/reference.conf

```diff
@@ -37,6 +37,18 @@ pekko.serialization.jackson {
 
 }
 
+#//#stream-read-constraints
+pekko.serialization.jackson {
+  read {
+    # see https://www.javadoc.io/static/com.fasterxml.jackson.core/jackson-core/2.15.2/com/fasterxml/jackson/core/StreamReadConstraints.html
+    # these defaults are the same as the defaults in `StreamReadConstraints`
+    max-nesting-depth = 1000
+    max-number-length = 1000
+    max-string-length = 20000000
```

Comment:
```
constraints {
    max-nesting-depth = 1000
    max-number-length = 1000
    max-string-length = 20000000
}
```
Seems more clear

Reply:
I honestly don't think adding the word `constraints` to the config names here makes them any clearer. Config names need to balance descriptiveness with conciseness.